wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Command.java

```diff
@@ -427,4 +487,36 @@ enum InterruptionBehavior {
     /** This command continues, and the incoming command is not scheduled. */
     kCancelIncoming
   }
+
+  static Command run(Runnable toRun, Subsystem... requirements) {
+    return new RunCommand(toRun, requirements);
+  }
+
+  static Command runOnce(Runnable toRun, Subsystem... requirements) {
```

Comment:
Perhaps rename to `instantly` to retain similarity to `InstantCommand`?

Reply:
I think `runOnce` is a clearer expression of intent and is fewer characters.  We might even want to consider renaming `InstantCommand` eventually?  "Instant" is a little ambiguous in that someone might read it as "starting without delay" instead of "lasting no duration."

Reply:
I think this could be a bit confusing, given how I'm not sure that running repeatedly is the default connotation of "run". But we have RunCommand, so idk.